docs/html/development/getting-started.rst

```diff
@@ -118,8 +118,27 @@ To build it locally, run:
 
 The built documentation can be found in the ``docs/build`` folder.
 
+
+REFERENCES(Helping Guide)
+-------------------------
+
+
+
 .. _`open an issue`: https://github.com/pypa/pip/issues/new?title=Trouble+with+pip+development+environment
+
+
 .. _`install Python`: https://realpython.com/installing-python/
 .. _`PEP 484 type-comments`: https://www.python.org/dev/peps/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code
 .. _`rich CLI`: https://docs.pytest.org/en/latest/usage.html#specifying-tests-selecting-tests
 .. _`GitHub`: https://github.com/pypa/pip
+.. _ `Tox` : https://tox.readthedocs.io/en/latest/example/basic.html
+.. _  `Pytest` : https://docs.pytest.org/en/latest/getting-started.html
+.. _ ` Mock` : https://realpython.com/python-mock-library/
+.. _ `pretend` : https://pypi.org/project/pretend/
+.. _ `version control` : https://www.atlassian.com/git/tutorials/what-is-version-control
+.. _ `Sphinx` : https://www.sphinx-doc.org/en/master/usage/quickstart.html
```

Comment:
The links to the projects' documentation isn't used anywhere and can be removed. (the :pypi:`tox` syntax links to the PyPI page for tox: https://pypi.org/project/tox/)